core/sentry_bot.py

```
@@ -0,0 +1,179 @@
+                await say(f"❌ Error handling Sentry issue: {str(e)}")
+
+        @self.app.message("debug error")
+        async def handle_error_debugging(message, say, context):
+            """Handle general error debugging requests"""
+            trace("slack.error_debugging_request", {
+                "user_id": message['user'],
+                "text": message['text'][:200]
+            })
+            try:
+                text = message['text']
+                user_id = message['user']
```

Ruff (0.12.2)

Local variable `user_id` is assigned to but never used

Remove assignment to unused variable `user_id`

(F841)